schema.sql

```diff
@@ -1,5 +1,8 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals (id SERIAL, 
+                      name TEXT, 
+                      date_of_birth DATE, 
+                      escape_attempts INT, 
+                      neutered BOOLEAN, 
+                      weight_kg NUMERIC);
```

Comment:
Hi @Ol-create 

### **OPTIONAL**
- [ ] Please try to change the data type for table `id` and `name` like so 
```sql
 id int,
    name varchar(255),
```

Reply:
# Hi 👋 Collins
I appreciate your effort
I had used SERIAL data type so that the id generation can be automated
Thank you I will try and change the name data type from TEXT to VARCHAR(225).
Thank you so much